src/settings_type.h

```diff
@@ -144,7 +145,7 @@ struct GUISettings {
 	uint8  scrollwheel_scrolling;            ///< scrolling using the scroll wheel?
 	uint8  scrollwheel_multiplier;           ///< how much 'wheel' per incoming event from the OS?
 	bool   timetable_arrival_departure;      ///< show arrivals and departures in vehicle timetables
-	bool   right_mouse_wnd_close;            ///< close window with right click
+	uint8   right_mouse_wnd_close;            ///< close window with right click
```

Comment:
Use the declared enum type?

Reply:
This wasn't done? Any reason why it was resolved and not done? :)